Fabric.Platform.Auth/AuthorizationMiddleware.cs

```diff
@@ -33,5 +34,28 @@ public static AppFunc Inject(AppFunc next, string[] requiredScopes, string[] all
                 return Task.CompletedTask;
             };
         }
+
+        private static bool HasPath(string[] allowedPaths, string pathToCheck)
+        {
```

Comment:
I think we can consolidate this into just 1 statement?

`return allowedPaths.Contains(pathToCheck)  || allowedPaths.Any(path => pathToCheck.StartsWith(path.TrimEnd('*'));`

Am I missing something? Would have to write some unit tests to confirm.